contrib/gitian-descriptors/gitian-linux-parallel.yml

```diff
@@ -0,0 +1,143 @@
+---
+name: "zcash-5.3.0-rc1"
```

Comment:
This doesn't seem to have been changed, as it should be, by `make-release.py`.

Reply:
Created a PR (https://github.com/zcash/zcash/pull/6214) to address this. I believe `make-release.py` was run before merging this PR https://github.com/zcash/zcash/pull/6104, so it was an old version of `make-release.py` which wasn't aware of the parallel gitian descriptor.